dashboard/src/pages/packages/index.vue

```diff
@@ -20,16 +20,16 @@ const { execute, error } = useAsyncState(() => {
   return packageStore.fetchPackages();
 }, null);
 
-const el = $ref<HTMLElement | null>(null);
+const el = ref<Element | null>(null);
 let tooltip: Tooltip | null = null;
 
 onMounted(() => {
-  if (el) tooltip = new Tooltip(el);
+  if (el.value) tooltip = new Tooltip(el.value);
 });
 
-let showLegend = $ref(false);
+let showLegend = ref(false);
 const toggleLegend = () => {
-  showLegend = !showLegend;
+  showLegend.value = !showLegend.value;
   if (tooltip) tooltip.hide();
```

Comment:
Why the `$ref` to `ref` change?

Reply:
I think `$ref` is outdated Vue syntax - the VS Code Vue.js linter complains about `$ref`.

Reply:
https://vuejs.org/guide/extras/reactivity-transform.html

Reply:
https://github.com/artefactual-sdps/enduro/issues/976

Reply:
@jraddaoui Yes, I've got the "Vue Official" VSC extension v2.0.22 and all the `$ref` instances are underlined in red. :confused: 

![image](https://github.com/artefactual-sdps/enduro/assets/626135/724ae76c-dd1f-4526-9221-e0aa82578d32)
![image](https://github.com/artefactual-sdps/enduro/assets/626135/523a8009-d452-402f-9571-785967c906f9)

Reply:
^ That's a screenshot on main

Reply:
I'd like to look at this better in #976 and address it across the entire project.

Reply:
> In any case, I'd leave it as it is for now, it's used in other places so we may want to create an issue to deal with it separately.

Do you mean leave it as it is in this PR (ref) or as in the main branch ($ref)?  I assumed the former.

Reply:
I mean like it was on main, so we can deal with it in #976.

Reply:
Done